samples/dotnet/graph-api-skills/appsettings.json

```diff
@@ -22,14 +22,14 @@
   "OneDrivePathToFile": "<path to a text file in your OneDrive>", // e.g. "Documents/MyFile.txt"
   "DefaultCompletionBackendLabel": "text-davinci-003",
   "OpenAI": {
-    "Label": "text-davinci-003",
-    "ModelId": "text-davinci-003",
-    "ApiKey": ""
+  //  "Label": "text-davinci-003",
+  //  "ModelId": "text-davinci-003",
+  //  "ApiKey": ""
```

Comment:
Undo?

Reply:
@shawncal the reason I commented these out is as follows:

1. Assume you want to run the sample and just use Azure OpenAI
2. The code checks is the label for the OpenAI configuration block and if not null will try to use it
3. This results in an error because there is no ApiKey, so to get the sample working you need to comment this out

If you still think I should undo this change I can do that

Reply:
Makes sense, thanks